jso/apis/src/main/java/org/teavm/jso/websocket/MessageEvent.java

```diff
@@ -0,0 +1,31 @@
+package org.teavm.jso.websocket;
+
+import org.teavm.jso.JSBody;
+import org.teavm.jso.JSObject;
+import org.teavm.jso.dom.events.Event;
+import org.teavm.jso.typedarrays.ArrayBuffer;
+
+public abstract class MessageEvent implements Event
+{
+    /** @return the data sent by the message emitter. */
+    @JSBody( script = "return this.data;" )
```

Comment:
Another possible way to express this is:

```java
@JsProperty
public abstract JsObject getData();
```

Reply:
I think it better reflects in the api to just have the property name for the method name when it represents access to an exposed property on the javascript object. Are you happy with this?